src/gui/sdlmain.cpp

```diff
@@ -3527,6 +3519,10 @@ static void GUI_StartUp(Section *sec)
 		            presentation_mode_pref.c_str());
 	}
 
+	const auto vfr_min_rate = section->Get_int("vfr_minimum_rate");
+	assert(vfr_min_rate > 0 && vfr_min_rate <= RefreshRateMax);
```

Comment:
Should this rather be an actual runtime check? In that case, we should document that valid values are in the 1 to 1000 Hz range, and if the provided value is outside of this range, we should default to 7 and log an error.

Reply:
conf setting is being knocked out